sdk/metricsadvisor/tests.yml

```diff
@@ -4,6 +4,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: metricsadvisor
+    TimeoutInMinutes: 120
```

Comment:
Increasing timeout because we were getting lots of "The job running on agent (...) ran longer than the maximum time of 60 minutes" errors.

Reply:
do u know why? is it because of the amount of data the service has it causes the service to take longer to return? or were u seeing the timeout before and you are fixing it also in this PR?

Reply:
They're separate issues. We've been getting the timeout in the last couple of days during nightly runs.